common/source/docs/common-compass-calibration-in-mission-planner.rst

```diff
@@ -4,15 +4,11 @@
 Compass Calibration
 ===================
 
-This article explains how to perform basic compass calibration.
-
-.. note::
-
-   This article assumes that you are using the most common configuration — a autopilot and compass mounted with the arrow on each facing toward the front of the vehicle. If you're using a different configuration see :ref:`Advanced Compass Setup <common-compass-setup-advanced>`.
+This article explains how to perform basic compass calibration. It assumes that you have at least one compass, either internally or externally in the system, and it has been enabled. See :ref:`Advanced Compass Setup <common-compass-setup-advanced>` for more information and to setup other compass related features.
 
 .. tip::
 
-   It is not necessary to recalibrate the compass when the vehicle is flown at a new location because ArduPilot includes a "declination database" which allows converting the location's magnetic North to true North without recalibrating.  In addition the location's "inclination" is calibrated at startup and then again soon after takeoff.
+   It is not necessary to recalibrate the compass when the vehicle is flown at a new location because ArduPilot includes a "declination database" which allows converting the location's magnetic North to true North without recalibrating if using the option "Obtain declination automatically".  In addition the location's "inclination" is calibrated at startup and then again soon after takeoff.
```

Comment:
instead of "declination database" use "world magnetic model"

Reply:
done..thanks